README.md

```diff
@@ -20,8 +20,6 @@ Add the dependency to any targets you've declared in your manifest:
 .target(name: "MyTarget", dependencies: ["Markdown"]),
 ```
 
-## Parsing
```

Comment:
Should we update the Copyright year to "2021-2022" in README.md?

Reply:
Good catch- thanks! Added here: https://github.com/apple/swift-markdown/pull/32/commits/f253ded8aa295aa75d6f8c173a2e96f290f2268b.